RELEASE_NOTES.txt

```diff
@@ -2,6 +2,194 @@ Apache Ignite Release Notes
 ===========================
 
 
+Apache Ignite In-Memory Computing Platform 2.9
+-----------------------------------------------------------
+
+Ignite Core:
+* Added feature of cluster snapshots for persistence caches
```

Comment:
Alex, I can't say for the docs completeness but would encourage us to run the text through Grammarly or a similar tool that can catch misprints.

For instance, we need to add the "a" or "the" article before "feature", "ability", etc. Also, Grammarly can help to rephrase some sentences.

Reply:
Thanks, fixed. Could you please have a look again?